contracts/src/LightClientArbitrum.sol

```diff
@@ -0,0 +1,15 @@
+// SPDX-License-Identifier: UNLICENSED
+
+pragma solidity ^0.8.0;
+
+import { LightClient } from "./LightClient.sol";
+
+interface ArbSys {
+    function arbBlockNumber() external view returns (uint256);
+}
+
+contract LightClientArbitrum is LightClient {
+    function currentBlockNumber() public view virtual override returns (uint256) {
```

Comment:
lovely! would it be possible to add some tests for this function?
Examples:
1.  show that block.number!=currentBlockNumber() in LightClientArbitrum 
2. currentBlockNumber() == Arbitrum's block number in LightClientArbitrum
3. calls to LightClient currentBlock is the L1 blocknumber

Reply:
Sounds good, will add these tests!

Reply:
https://github.com/EspressoSystems/espresso-sequencer/pull/2423/commits/56ab328fe9068131a6edac67658b28d8b9bc48b9